<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="6">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item row="1" column="0" colspan="6">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelPort">
      <property name="text">
       <string>串口号</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="labelRate">
      <property name="text">
       <string>波特率</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="labelData">
      <property name="text">
       <string>数据位</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="labelCheck">
      <property name="text">
       <string>校验位</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="labelStop">
      <property name="text">
       <string>停止位</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QPushButton" name="pushButtonSend">
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QComboBox" name="comboBoxPort"/>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="comboBoxRate"/>
    </item>
    <item row="3" column="2">
     <widget class="QComboBox" name="comboBoxData"/>
    </item>
    <item row="3" column="3">
     <widget class="QComboBox" name="comboBoxCheck"/>
    </item>
    <item row="3" column="4">
     <widget class="QComboBox" name="comboBoxStop"/>
    </item>
    <item row="3" column="5">
     <widget class="QPushButton" name="pushButtonOpen">
      <property name="text">
       <string>打开串口</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
